roles/openshift_docker_gc/templates/dockergc-ds.yaml.j2

```diff
@@ -23,9 +23,9 @@ items:
         name: dockergc
       spec:
 {# Only set nodeSelector if the dict is not empty #}
-{% if r_docker_gc_node_selectors %}
+{% if openshift_crio_docker_gc_node_selector %}
```

Comment:
This if looks to be unnecessary. We're defining this value by default, so I suppose it should always populate the nodeSelector value.